requirements/edx/testing.txt

```diff
@@ -756,9 +753,8 @@ idna==3.4
     #   yarl
 import-linter==1.6.0
     # via -r requirements/edx/testing.in
-importlib-metadata==5.1.0
+importlib-metadata==6.0.0
```

Comment:
One major change in the new version https://github.com/python/importlib_metadata/blob/main/CHANGES.rst#v600 but it should be okay to upgrade the package.